code/src/entrance.c

```diff
@@ -88,6 +88,13 @@ void Scene_Init(void) {
 
     gRestrictionFlags[72].flags3 = 0; // Allows farore's wind in GTG
     gRestrictionFlags[94].flags3 = 0; // Allows farore's wind in Ganon's Castle
+    // Allow Farore's Wind anywhere if the option is enabled
+    // Cannot be used in areas affected by a general item restriction
+    if (gSettingsContext.faroresWindAnywhere){
+        for (int i = 0; i < 99; i++){
+			gRestrictionFlags[i].flags3 &= ~0x30;
+		}
+    }
```

Comment:
I think this code block was intentionally removed in a previous commit, but I dont want to say definitively until @HylianFreddy can double check

Reply:
Yes that's correct. The FW anywhere setting is handled by a separate function, so changing the restriction flags isn't needed